nss-day-cohort-30/bangazon-site-suave-salmon
# Product categories dk

Fixes:
N/a
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
Added a product categories view. This uses a new controller and view (types) to grab each category type and list the first 3 products of the category type
-

Steps to test:
click product categories in nav
click category and should be redirected to full category page
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
N/a
-

Commit message:
added product categories view

Link to feature ticket:
-

@suave-salmon
